### PR TITLE
feat: add react-prisma

### LINKED ISSUE #4767: Implement `react-prisma` prototype
In the same fashion as `react-pg`, we should provide a `react-prisma` package, that supports React suspense and therefore React Server Components https://github.com/facebook/react/tree/master/packages/react-pg

### PR DESCRIPTION
**Pull request by timsuchanek**
Closes https://github.com/prisma/prisma/issues/4767

### CODE REVIEW

File: src/packages/react-prisma/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "react-prisma",
+  "version": "0.0.5",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
+  "author": "Tim Suchanek <suchanek@prisma.io>",
```
Comment (Jolg42) [inline src/packages/react-prisma/package.json:7]:
```suggestion
  "author": "Tim Suchanek <suchanek@prisma.io>",
  "repository": "git@github.com:prisma/prisma.git",
```

File: src/packages/react-prisma/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  ],
+  "lint-staged": {
+    "*.ts": [
+      "eslint",
+      "prettier --write"
+    ]
+  },
+  "peerDependencies": {
+    "react": "^17.0.0",
+    "@prisma/client": "*"
```
Comment (Jolg42) [inline src/packages/react-prisma/package.json:48]: maybe >=2.13.0 or something like that?

Comment (timsuchanek) [reply src/packages/react-prisma/package.json:48]: There is no constraint necessary, as we didn't need to change anything in the client.

Comment (Jolg42) [reply src/packages/react-prisma/package.json:49]: But it relies on dmmf stuff so it's likely broken in older versions I guess

Comment (timsuchanek) [reply src/packages/react-prisma/package.json:49]: The DMMF has always been exported like that.

Comment (Jolg42) [reply src/packages/react-prisma/package.json:49]: Ok, I would at least put >=2.0.0 especially with the prisma npm switch if it happens though.

**Pull request merged**